pairing/bn256/point.go

```diff
@@ -62,7 +62,34 @@ func (p *pointG1) Clone() kyber.Point {
 }
 
 func (p *pointG1) EmbedLen() int {
-	panic("bn256.G1: unsupported operation")
+	return (255 - 8 - 8) / 8
+}
+
+func gfpToBig(gfp *gfP) *big.Int {
+	bigInt := new(big.Int)
+	decoded := new(gfP)
+	montDecode(decoded, gfp)
+	buf := make([]byte, 32)
+	decoded.Marshal(buf)
+	bigInt.SetBytes(buf)
+	return bigInt
+}
+
+func bigToGfp(bigX *big.Int) *gfP {
```

Comment:
Could you move this to a function in `gfp.go` ? Something like `newGFpFromBigInt(...)`

Reply:
Another good idea, this should also be in.